system/ee/installer/updates/ud_4_00_05.php

```diff
@@ -44,7 +44,7 @@ public function do_update()
     private function fixGridModifierParsing()
     {
         $grid_fields = ee('Model')->get('ChannelField')
-            ->fields('field_id')
+            ->fields('field_id', 'legacy_field_data')
```

Comment:
@intoeetive So What is this actually solving?

Reply:
`$field->getDataStorageTable(),` is using `legacy_field_data`, so it needs to be fetched to make things work properly.

@dougblackjr has confirmed this fix works for him